pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 python = "^3.8.1"
 blessed = "^1.20.0"
 readchar = "^4.0.6"
-python-editor = "^1.0.4"
+editor = "^1.6.0"
```

Comment:
Can you change it to the latest version? `editor = "^1.6.6"`

To make sure we are not using old versions with bugs...

Thanks!

Reply:
done, cheers.